CONTRIBUTING.md

```diff
@@ -1 +1,7 @@
-Please see [Contribution Guidelines](https://developers.mattermost.com/contribute/getting-started/)
+# "Your contribution will help the world💖"
```

Comment:
This doesn't seem like the style that would be used in a technical file like `CONTRIBUTING.md`. It's a bit "over the top". It's fair to say which impact open source contributions have, but that would be more fitting on a page like https://mattermost.com/contribute/